ibex.core

```diff
@@ -28,7 +28,6 @@ filesets:
 #      - rtl/ibex_register_file.sv # ASIC
       - rtl/ibex_register_file_ff.sv # FPGA
       - rtl/ibex_core.sv
-      - rtl/include/ibex_config.sv: { is_include_file: true }
```

Comment:
Don't you still need the ibex_config.sv file here? But not as an include.

Reply:
ibex_config.sv is completely gone now